### PR TITLE
[Bug Fix] Fix the bug reported by RK3568 Runtime under the latest driver

### PR DESCRIPTION
**Pull request by Zheng-Bicheng**
### PR types(PR类型)
Bug Fix

### Description
Fix the bug reported by RK3568 Runtime under the latest driver

### CODE REVIEW

**Comment by Zheng-Bicheng** [comment]
修复RK3568在最新驱动下报错的BUg

File: fastdeploy/runtime/backends/rknpu2/rknpu2_backend.cc [hunk 1]
```diff
@@ -67,6 +67,9 @@ bool RKNPU2Backend::GetSDKAndDeviceVersion() {
  ***************************************************************/
 void RKNPU2Backend::BuildOption(const RKNPU2BackendOption& option) {
   this->option_ = option;
+  std::cout << "option.cpu_name is " << option.cpu_name << std::endl;
+  std::cout << "option.core_mask is " << option.core_mask << std::endl;
```
Comment (jiangjiajun) [inline fastdeploy/runtime/backends/rknpu2/rknpu2_backend.cc]: 删除debug日志

File: fastdeploy/runtime/backends/rknpu2/rknpu2_backend.cc [hunk 2]
```diff
@@ -101,6 +104,7 @@ bool RKNPU2Backend::InitFromRKNN(const std::string& model_file,
 
   // SetCoreMask if RK3588
   if (this->option_.cpu_name == rknpu2::CpuName::RK3588) {
+    std::cout << "Device is RK3588" << std::endl;
```
Comment (jiangjiajun) [inline fastdeploy/runtime/backends/rknpu2/rknpu2_backend.cc]: 同上

**Review by jiangjiajun — CHANGES_REQUESTED** [review]

**Comment by Zheng-Bicheng** [comment]
@jiangjiajun Done.

**Review by jiangjiajun — APPROVED** [review]

**Pull request merged**